src/main/java/com/hivemq/cli/commands/cli/PublishCommand.java

```diff
@@ -115,7 +119,13 @@ public void run() {
             if (isVerbose()) put("writer1.level", "trace");
         }};
 
-        LoggerUtils.useDefaultLogging(configurationMap);
+        if (logToLogfile) {
```

Comment:
This is duplicated and could be moved to a util. e.g. LoggerUtils.setupLogging(logToLogfile)